src/rfcml_bending/bending.py

```diff
@@ -203,7 +211,7 @@ def bending_mode_correction(
 
             modes = list(range(0, n_modes))
             rmsForces, fit, forces = self.fit_bending(modes, fea_map)
-        elif method == "orig":
+        elif method == "orig": # orig method ONLY works with RW forcespace matrices
```

Comment:
should we just get rid of this now? Is there any reason to conserve the use of the old forcespace we don't understand?

Reply:
I think we could drop modalCorrection_orig as well